Mathlib/CategoryTheory/Adjunction/Unique.lean

```diff
@@ -35,89 +26,9 @@ variable {C D : Type*} [Category C] [Category D]
 
 namespace CategoryTheory.Adjunction
 
-/--
-If `F ⊣ G` and `F' ⊣ G'` are adjunctions, then giving a natural transformation `G ⟶ G'` is the
-same as giving a natural transformation `F' ⟶ F`.
--/
-@[simps]
-def natTransEquiv {F F' : C ⥤ D} {G G' : D ⥤ C} (adj1 : F ⊣ G) (adj2 : F' ⊣ G') :
```

Comment:
Do you think it's worth deprecating these (in favour of the conjugateIso ones) first, rather than straight-up deleting them first?

Reply:
They have only existed for a few months, so I think it's fairly unlikely that they are being used outside mathlib. But I'm happy with whatever

Reply:
I've added aliases for the `Equiv`s `natTransEquiv` and `natIsoEquiv`. They don't match `conjugateEquiv` and `conjugateIsoEquiv` exactly (they go in the other direction), so I didn't add any for the API lemmas